oeps/oep-0001.rst

```diff
@@ -289,6 +298,12 @@ ReStructuredText [8] allows for rich markup that is relatively easy to read,
 and can also be rendered into good-looking and functional HTML. OEPs are
 rendered to HTML using Sphinx. An `OEP template`_ can be found in the repo.
 
+OEPs may be discussed in a more convenient format, such as a Google Doc, if
+it is deemed appropriate for the audience and sufficiently open to comment and
+review. The final OEP should be committed as described above with a link to the
```

Comment:
We really need the likely proposers of "significant product enhancements" to weigh in on whether this process will work for them.  I understand the value of a single auditable trail of information. But will the people designing the enhancements use this process?